index.bs

```diff
@@ -94,7 +94,7 @@ and a <dfn for="file entry">shared lock count</dfn> (a number representing the n
 
 <div algorithm>
 To <dfn for="file entry/lock">take</dfn> a [=file entry/lock=] with a |value| of "`exclusive`" or "`shared`" on a given [=file entry=] |file|,
-run the following steps:
+run these steps:
```

Comment:
@domenic has been encouraging me to leave of "run these steps" entirely, when they occur at the end. I think it works well, especially for sentences starting with "To".

Reply:
Ah nice, that sounds good to me